src/Selenium2Library/keywords/_element.py

```diff
@@ -680,6 +679,31 @@ def remove_location_strategy(self, strategy_name):
         """
         self._element_finder.unregister(strategy_name)
 
+    def element_attribute_should_be(self, locator, attr):
+        """Checks whether element has attribute with value
+
+        Usage Example:
+        | Element Attribute Should Be | xpath=//input | key=value |
+        """
+        self._info("Checking attributes '%s' for element '%s'" % (
+                                                    attr, locator))
+        try:
+            key, value = attr.split("=", 1)
+        except ValueError:
+            raise AssertionError(
+                    "Attribute %s does not match pattern key=value" % attr)
+
+        el = self._element_find(locator, True, True)
+        retval = el.get_attribute(key)
+
+        if retval is None:
```

Comment:
This needs thinking, is different error message required if attribute value is empty. Keyword will fail also in 703 if this if is removed. But it prevents to check is attribute value empty. Did you have some use case for this?